opentracing-cdi/src/main/java/io/opentracing/contrib/cdi/Traced.java

```diff
@@ -0,0 +1,18 @@
+package io.opentracing.contrib.cdi;
+
+
+import javax.interceptor.InterceptorBinding;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Inherited
+@InterceptorBinding
+@Retention(RUNTIME)
+@Target({METHOD, TYPE})
+public @interface Traced {
```

Comment:
Missing `JavaDoc` on the annotation.
This will become an important (visible) part of the API.